src/Features/Core/Portable/CodeFixesAndRefactorings/AbstractFixAllCodeAction.cs

```diff
@@ -53,33 +54,30 @@ public override string Title
 
         internal override string Message => FeaturesResources.Computing_fix_all_occurrences_code_fix;
 
-        protected sealed override async Task<IEnumerable<CodeActionOperation>> ComputeOperationsAsync(CancellationToken cancellationToken)
-            => await ComputeOperationsAsync(new ProgressTracker(), cancellationToken).ConfigureAwait(false);
-
-        internal sealed override Task<ImmutableArray<CodeActionOperation>> ComputeOperationsAsync(
-            IProgressTracker progressTracker, CancellationToken cancellationToken)
+        protected sealed override Task<ImmutableArray<CodeActionOperation>> ComputeOperationsAsync(
```

Comment:
Wy is `Task<IEnumerable<CodeActionOperation>> ComputeOperationsAsync(CancellationToken cancellationToken)` removed?

Reply:
nvm, I see what's going on